app/models/audit_activity/alert/add.rb

```diff
@@ -1,25 +1,17 @@
 class AuditActivity::Alert::Add < AuditActivity::Base
-  include ActivityNotification
   extend ActionView::Helpers::NumberHelper
   belongs_to :investigation
 
-  def self.from(alert)
-    create!(
-      title: "Product safety alert sent",
-      body: build_body(alert),
-      source: UserSource.new(user: User.current),
-      investigation: alert.investigation
-    )
+  def self.build_metadata(alert, user_count)
+    {
+      user_count: user_count,
+      subject: alert.summary,
+      date_sent: alert.created_at,
+      description: alert.description
+    }
```

Comment:
I think we would want to store an `Integer` here - I think at the moment it would store a string passed through from the controller. I don't think we need to pass through the count at all.

```suggestion
  def self.build_metadata(alert)
    {
      user_count: User.active.count,
      subject: alert.summary,
      date_sent: alert.created_at,
      description: alert.description
    }
```